### DISCUSSION TITLE
space

### DISCUSSION THREAD

**Discussion by uriasjaime515** [fontforge/fontforge#4887] [Q&A]
i created a new font and am having issues actually locatinf the glyph for the spacebar i have looked around at others answers and read the issue others had and seen the fix but i still dont understand where the location of the spacebar  is can anyone help me out

**Comment by mktcode** [comment]
Just hit the spacebar (you can just type the "character" you are looking for) and it will be marked in the overview. Is that what you need?

![image](https://user-images.githubusercontent.com/6792578/149840440-ff9428fd-4223-4949-837e-a232e6200728.png)

Disclaimer: Today is the first time I ever used such a tool. I'm a dev, not a typographer. :D
